modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/ext/OpenAPIExtension.java

```diff
@@ -27,4 +27,14 @@ ResolvedParameter extractParameters(List<Annotation> annotations, Type type, Set
      * @param chain     the chain with swagger extensions to process
      */
     void decorateOperation(Operation operation, Method method, Iterator<OpenAPIExtension> chain);
+
+    /**
+     * Decorates operation with additional vendor based extensions.
+     *
```

Comment:
would add some clarification in javadocs, mentioning that new overload is available `@since 2.1.9` and that the two overloads are both invoked in a given order